### DISCUSSION TITLE
how do I install SQLiteStudio on a chromebook?

### DISCUSSION THREAD

**Discussion by gregoireleveaux** [pawelsalawa/sqlitestudio#4196] [Q&A]
Hello,
I would like to start using this app on my chromebook but after a day of research I still don't know the procedure ...
If anyone knows the method I would be very grateful!
Have a great day,
Greg

**Comment by pawelsalawa** [comment]
Is chromebook able to use other Linux/Windows binaries?

**Comment by gregoireleveaux** [comment]
Yes, I am using the Linux option for developers on my Chromebook.

**Reply by pawelsalawa** [reply]
So... have you tried running Linux binary of SQLiteStudio? https://github.com/pawelsalawa/sqlitestudio/releases/download/3.3.3/sqlitestudio-3.3.3.tar.xz

**Comment by gregoireleveaux** [comment]
I unzipped the file. I now have an SQLiteStudio3 folder in my Linux folder. But then I don't know what to do ...
In any case, thank you for your patience!

**Reply by pawelsalawa** [reply]
Inside of SQLiteStudio3 folder there is the `SQLiteStudio` binary file. You need to run it (double click? I don't know how that works on Chromebook).

**Reply by apdave11** [reply]
Hi Pawel. I'm a novice. Please provide or refer me to the step-by-step instructions for running SQLiteStudio on Linux for the portable file and installing on Linux for the installer file. I've tried multiple times and ways in a Linux environment that is running on ChromeOS and have been unsuccessful.

**Reply by pawelsalawa** [reply]
1. Download https://github.com/pawelsalawa/sqlitestudio/releases/download/3.4.4/sqlitestudio-3.4.4.tar.xz
2. Decompress downloaded file, so you get the `SQLiteStudio` directory.
3. Enter that directory.
4. Execute SQLiteStudio binary file (`sqlitestudio`) from that directory.

**Reply by apdave11** [reply]
I don't know how to do the steps above, but I kept searching online and did some guessing and was able to launch SQLiteStudio with the following steps:

1. In the chromebook that runs ChromeOS, went into settings and set up the Linux development environment.
2. Went to the webpage below and downloaded the installer file for Linux x64 into the "Linux files" folder. 
     https://github.com/pawelsalawa/sqlitestudio/releases
3. Opened the Terminal app and selected "penguin" to open a window.
4. In the terminal window, entered the following sequence of commands and selections:
sudo chmod 755 SQLiteStudio-3.4.4-linux-x64-installer.run
sudo ./SQLiteStudio-3.4.4-linux-x64-installer.run
Scroll through the License Agreement by pressing "Enter" multiple times. 
Accept the License Agreement: y
Install the application for [1] all users or [2] local user only: 1
Associate SQLite file extensions with SQLiteStudio: Y
Specify the Installation Directory [/opt/SQLiteStudio]: /opt/SQLiteStudio
Do you want to continue? Y
Launch SQLiteStudio now. Y

**Reply by pawelsalawa** [reply]
Seems like the installation went okay. If it doesn't run, you can try running it by yourself from some kind of "Start" menu, or something. I cannot help you further, cause I'm completely unfamiliar with ChromeOS.

**Comment by gregoireleveaux** [comment]
Here is what I tried without success :
![Screenshot 2021-08-26 20 42 49](https://user-images.githubusercontent.com/73223421/131018513-7d8964ad-a1bd-4b92-89c7-26ea826eff68.png)

**Reply by pawelsalawa** [reply]
You've downloaded a source code package. Have you clicked on the link I posted above?

**Reply by gregoireleveaux** [reply]
Whoops !
Try again (trying to rename problematic files):

![Screenshot 2021-08-28 19 52 12](https://user-images.githubusercontent.com/73223421/131226621-26d17f81-c818-4803-b01b-48030a2a783a.png)

**Reply by pawelsalawa** [reply]
Looks like it doesn't run linux application that well. It did not manage to load files properly.
Try to set `LD_LIBRARY_PATH` environment variable to include `./lib` and then re-try.
I don't know chromebook unfortunetly, so I don't have other suggestions for you. Maybe you can try on the superuser.com or some other forum dedicated to ChromeBook users.

**Comment by gregoireleveaux** [comment]
It is beyond my skills and I feel like chromebooks are not sufficiently advanced in the management of linux applications.
But really thank you for taking the time to answer me.
